common/scripted_triggers/wc_maa_triggers.txt

```diff
@@ -201,10 +201,12 @@ has_special_aerial_maa_trigger = {
 		has_goblin_heritage_trigger = yes
 		this = culture:blood_elf
 		this = culture:dryad
+		this = culture:durotaric
 		this = culture:high_elf
 		this = culture:naga
 		this = culture:night_elf
 		this = culture:scourge
+		this = culture:wildhammer
```

Comment:
What aerial MaA they have?

Reply:
Durotaric - Wyvern
Wildhammer- Gryphon Rider

Reply:
But they are regional.

Reply:
Wildhammer has it from the start
AFAIK Durotaric will only exist in the wyvern area anyway and unlock the maa

Reply:
What this trigger represents? Cultures that have unique MaAs or cultures that unlocked MaAs innovations?
If the second, it should check for `has_innovation`.

Reply:
By name and functionality of the trigger, it's the first case.

Reply:
Yes the first case, it's not who has it available, but who has it unlocked at the start.

Reply:
> Yes the first case, it's not who has it available, but who has it unlocked at the start.

The Wildhammer culture has it from the start, but it's not unique, it's regional.

Reply:
The first case implies it's trigger for cultures that can unlock/unlocked unique cultural.

Reply:
If you need to check if culture unlocked any MaA innovation, you should check `has_innovation` and prob need another trigger.

Reply:
But I don't know where you may need it. I don't see such cases.

Reply:
> Member



> > Yes the first case, it's not who has it available, but who has it unlocked at the start.
> 
> The Wildhammer culture has it from the start, but it's not unique, it's regional.

Its just checking for an unlocked MaA regardless of regional or cultural, if a culture has an MaA it should get the building

Reply:
> The first case implies it's trigger for cultures that can unlock/unlocked unique cultural innovations.

Only has unlocked, past tense. None of the cultures listed are missing an MaA

Reply:
> If you need to check if culture unlocked any MaA innovation, you should check `has_innovation` and prob need another trigger.

![image](https://user-images.githubusercontent.com/74790101/188338202-cccc1028-10c5-4a19-a026-58d4cd40162a.png)
The first part checks if there is any basic MaA innovation, then it checks the list for cultures that already have MaA unlocked at the start. It would take up extra time and space using a has_innovation for each of the special MaA of the cultures listed